.github/workflows/sample-run.yml

```diff
@@ -22,4 +22,23 @@ jobs:
           go build -v ./...
 
       - name: Build arm 
-        run: GOOS=linux GOARCH=arm64 go build -v ./...
\ No newline at end of file
+        run: |
+          GOOS=linux GOARCH=arm64 go build -v ./...
+
+      - name: Run Tests
+        run: |
+          go test -v ./...   # This runs all the tests in the project and provides detailed output.
+
+      - name: Display Test Logs
+        if: always()  #  runs regardless whether the tests pass or fail.
+        run: |
+          echo "Displaying logs from tests..."
+          go test -v ./... | tee test-output.log
```

Comment:
You don't need to use go test, you need to show the logs from the keploy test command that is given in the shell scripts.